[2024-11-23T18:13:37.999+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-23T18:13:38.237+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fraud_detection.fetch_api scheduled__2024-11-23T18:12:00+00:00 [queued]>
[2024-11-23T18:13:38.288+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fraud_detection.fetch_api scheduled__2024-11-23T18:12:00+00:00 [queued]>
[2024-11-23T18:13:38.289+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-11-23T18:13:38.444+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): fetch_api> on 2024-11-23 18:12:00+00:00
[2024-11-23T18:13:38.488+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=4877) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-11-23T18:13:38.500+0000] {standard_task_runner.py:72} INFO - Started process 4884 to run task
[2024-11-23T18:13:38.496+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'fraud_detection', 'fetch_api', 'scheduled__2024-11-23T18:12:00+00:00', '--job-id', '486', '--raw', '--subdir', 'DAGS_FOLDER/fraud_detection_dag.py', '--cfg-path', '/tmp/tmpt56jacbu']
[2024-11-23T18:13:38.524+0000] {standard_task_runner.py:105} INFO - Job 486: Subtask fetch_api
[2024-11-23T18:13:38.795+0000] {task_command.py:467} INFO - Running <TaskInstance: fraud_detection.fetch_api scheduled__2024-11-23T18:12:00+00:00 [running]> on host 345e92ff26bf
[2024-11-23T18:13:38.950+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='fraud_team' AIRFLOW_CTX_DAG_ID='fraud_detection' AIRFLOW_CTX_TASK_ID='fetch_api' AIRFLOW_CTX_EXECUTION_DATE='2024-11-23T18:12:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-11-23T18:12:00+00:00'
[2024-11-23T18:13:38.952+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-23T18:13:42.853+0000] {fraud_detection_dag.py:65} INFO - Transaction reçue avec succès
[2024-11-23T18:13:42.958+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-11-23T18:13:43.031+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-23T18:13:43.032+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=fraud_detection, task_id=fetch_api, run_id=scheduled__2024-11-23T18:12:00+00:00, execution_date=20241123T181200, start_date=20241123T181338, end_date=20241123T181343
[2024-11-23T18:13:43.230+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-23T18:13:43.397+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-11-23T18:13:43.414+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2024-11-23T18:13:43.588+0000] {standard_task_runner.py:217} INFO - Process not found (most likely exited), stop collecting metrics
